react/modules/useButtonClasses.ts

```diff
@@ -0,0 +1,82 @@
+// This code comes form the vtex.store-link hook
+// to add calsses of the styleguide button to a
+// element. We might have a component to share
+// this code between apps in a future
+// https://github.com/vtex-apps/store-link/blob/master/react/modules/useButtonClasses.ts
+
+import { useMemo } from 'react'
+import classnames from 'classnames'
+
+// for now this are the only necessary variant and sizes
+type Size = 'regular'
+type Variant = 'secondary'
+
+interface ButtonProps {
+  variant: Variant
+  size: Size
+}
+
+interface AllOptions {
+  variant: ButtonProps['variant']
+  disabled: boolean
+  size: ButtonProps['size']
+}
+
+type Options = Partial<AllOptions>
+
+const SECONDARY_DISABLED = ''
+const SECONDARY_ENABLED =
+  'bg-action-secondary b--action-secondary c-on-action-secondary hover-bg-action-secondary hover-b--action-secondary hover-c-on-action-secondary'
+
+const BASE_CONTAINER_CLASSES = 'bw1 fw5 ba v-mid pa0 lh-solid br2 pointer'
+const LABEL_BASE = 'w-100 tc'
+
+const REGULAR = 'min-h-regular t-action'
+
+const LABEL_REGULAR = 'ph6'
+
+function getClassesByVariant(
+  variant: ButtonProps['variant'],
+  disabled: boolean
+) {
+  let container
+
+  // eslint-disable-next-line default-case
+  switch (variant) {
+    case 'secondary':
+      container = disabled ? SECONDARY_DISABLED : SECONDARY_ENABLED
+  }
```

Comment:
```suggestion
  switch (variant) {
    case 'secondary':
      container = disabled ? SECONDARY_DISABLED : SECONDARY_ENABLED
      break
    // no default
  }
```

Reply:
I do agree with the suggestion from @kaisermann , but also, why isn't there a default? Event if the behavior were to be the same, I think it would be good to have a `default` for this `switch`. Also, in the future, as we add more cases here, the existence of this `default` could prevent weird bugs.

Reply:
> I do agree with the suggestion from @kaisermann , but also, why isn't there a default? Event if the behavior were to be the same, I think it would be good to have a `default` for this `switch`. Also, in the future, as we add more cases here, the existence of this `default` could prevent weird bugs.

`container` is the variable that has the classes that will be added to the container element, what would be the default case here is that we won't add any classes which mean that the default case would be something like `default: container = undefined` which doesn't make any sense, because `container` already has the value of `undefined` in it